src/main/java/com/github/edouardswiac/zerotier/api/ZTCRule.java

```diff
@@ -0,0 +1,70 @@
+package com.github.edouardswiac.zerotier.api;
+
+public final class ZTCRule {
+  int ruleNo;
```

Comment:
since there are getters/setters for those fields I'd suggest we set them private